### PR TITLE
Add glyph's docstring and test

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

I checked `glyph` method to use my project. I found it the befavior is correct 👍   I added small fix of docstring and test of `glyph`.

 

### Details

- Add explicit explanation of `orientation` and `scale` parameter.
- Add a test to check glyph is oriented and scaled correctly.

### CODE REVIEW

**Review by tkoyama010 — COMMENTED** [review]
WFM

File: pyvista/core/filters/data_set.py [hunk 1]
```diff
@@ -1577,10 +1577,12 @@ def glyph(self, orient=True, scale=True, factor=1.0, geom=None,
         orient : bool or str, optional
             If ``True``, use the active vectors array to orient the glyphs.
             If string, the vector array to use to orient the glyphs.
+            If ``False``, orient of the glyphs is off.
```
Comment (akaszynski) [inline pyvista/core/filters/data_set.py:1580]:
```suggestion
            If ``False``, the glyphs will not be orientated.
```

File: pyvista/core/filters/data_set.py [hunk 2]
```diff
@@ -1577,10 +1577,12 @@ def glyph(self, orient=True, scale=True, factor=1.0, geom=None,
...
 
         scale : bool, str or sequence, optional
             If ``True``, use the active scalars to scale the glyphs.
             If string, the scalar array to use to scale the glyphs.
+            If ``False``, scale of the glyphs is off.
```
Comment (akaszynski) [inline pyvista/core/filters/data_set.py:1585]:
```suggestion
            If ``False``, the glyphs will not be scaled.
```

**Review by akaszynski — APPROVED** [review]
I made two minor changes.  Otherwise LGTM.

**Pull request merged**